use-cases/response-transform-utils.js

```diff
@@ -0,0 +1,64 @@
+const {estimatedDiameterInMeters} = require('../app-utils/iso-utils');
+
+const transformMeteorResponse = (response) => {
+
+  const responseBody = {};
+
+  for (const key in response.data.near_earth_objects) {
+    responseBody[key] = [];
+    if (response.data.near_earth_objects.hasOwnProperty(key)) {
+
+      response.data.near_earth_objects[key].forEach(meteor => {
+            responseBody[key].push(
+                {
+                  id: meteor.id,
+                  name: meteor.name,
+                  average_diameter: estimatedDiameterInMeters(
+                      meteor.estimated_diameter.meters.estimated_diameter_min,
+                      meteor.estimated_diameter.meters.estimated_diameter_max
+                  ),
+                  is_potentially_hazardous_asteroid: meteor.is_potentially_hazardous_asteroid,
+                  close_approach_data: meteor.close_approach_data,
+                  relative_velocity: meteor.close_approach_data[0].relative_velocity.kilometers_per_second
+                }
+            )
+          }
+      );
+    }
+  }
```

Comment:
Just my thoughts - good idea to separate return statement with empty line - easier to read